purchase_fop_shipping/models/partner.py

```diff
@@ -10,4 +10,5 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     fop_shipping = fields.Float(
-        'FOP shipping', help='Min purchase order amount for FOP shipping',)
+        'FOP shipping', help='Min purchase order amount for FOP shipping',
+        company_dependent=True)
```

Comment:
@Cedric-Pigeon Shouldn't have migration script for the current company (will data be kept) ?

Reply:
@rousseldenis You are right. Data are lost by how can we know the related company in a migration script? Not sure the current one is the best option.

Reply:
So, a comment in readme for that version ?

Reply:
@rousseldenis Done